pkg/collector/corechecks/sbom/check.go

```diff
@@ -173,13 +180,25 @@ func (c *Check) Run() error {
 			c.processor.processContainerImagesRefresh(c.workloadmetaStore.ListImages())
 		case <-hostPeriodicRefreshTicker.C:
 			c.processor.processHostRefresh()
+		case <-metricTicker.C:
+			c.sendUsageMetrics()
 		case <-c.stopCh:
 			c.processor.stop()
 			return nil
 		}
 	}
 }
 
+func (c *Check) sendUsageMetrics() {
+	c.sender.Count("datadog.agent.sbom.container_images.running", 1.0, "", nil)
+
+	if ddConfig.Datadog.GetBool("sbom.host.enabled") {
+		c.sender.Count("datadog.agent.sbom.hosts.running", 1.0, "", nil)
```

Comment:
The value of this metric never changes. Would it make sense to use agent config inventory feature for this?